apps/api/src/app/controllers/utilities/mixTagsIntoViewModel.ts

```diff
@@ -0,0 +1,27 @@
+import { isDeepStrictEqual } from 'util';
+import { ResourceTypeOrNoteType } from '../../../domain/models/categories/types/ResourceTypeOrNoteType';
+import { Tag } from '../../../domain/models/tag/tag.entity';
+import { TagViewModel } from '../../../view-models/buildViewModelForResource/viewModels';
+import { BaseViewModel } from '../../../view-models/buildViewModelForResource/viewModels/base.view-model';
+
+export default (
```

Comment:
It might be nice to unit-test this, since we are only using snapshots to validate the tags in the query response.

Reply:
We need to finalize the api first. I added a unit-test as part of the requirements of [this story](https://www.pivotaltracker.com/story/show/182202065).